### PR TITLE
Add readme for podspec presubmit tests

### PR DESCRIPTION
**Pull request by granluo**

### CODE REVIEW

File: scripts/create_spec_repo/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# Podspec presubmit test setup
+
+Podspec presubmit test is to help ensure podspec is releasable. 'pod spec lint' will run for SDKs with sources of
+
+- https://github.com/firebase/SpecsTesting
+- https://github.com/firebase/SpecsDev.git
+- https://github.com/firebase/SpecsStaging.git
+- https://cdn.cocoapods.org/
+
+where [SpecsTesting](https://github.com/firebase/SpecsTesting) is generated from the head of the master branch of [firebase-ios-sdk repo](https://github.com/firebase/firebase-ios-sdk).
```
Comment (paulb777) [inline scripts/create_spec_repo/README.md:10]: nit: Please limit line lengths to 100 to keep readable in any editor

File: scripts/create_spec_repo/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    steps:
+    - uses: actions/checkout@v2
+    - name: Setup Bundler
+      run: scripts/setup_bundler.sh
+    - name: Build and test
+      run: scripts/third_party/travis/retry.sh pod spec lint FirebaseDatabase.podspec --skip-tests --sources='https://github.com/firebase/SpecsTesting','https://github.com/firebase/SpecsDev.git','https://github.com/firebase/SpecsStaging.git','https://cdn.cocoapods.org/'
+
+```
+
+Merging in postsubmit is not required to set up, [`update_SpecsTesting_repo` job](https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/prerelease.yml#L48) in the [prerelease workflow](https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/prerelease.yml) will automatically `pod repo push` changed podspecs.
```
Comment (paulb777) [inline scripts/create_spec_repo/README.md:36]: Doesn't there need to be a `pod repo push` in postsubmit to support multiple merging PRs between the nightly prerelease.yml runs?

Comment (granluo) [reply scripts/create_spec_repo/README.md:36]: Yes, and we have had a [job](https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/prerelease.yml#L48-L94) to update a podspec repo in post submit between nightly updates. By 'not required', I am meaning users who intend to set up a job do not need to worry about postsubmits. Will rephrase to make it more clear.

File: scripts/create_spec_repo/README.md [hunk 3]
```diff
@@ -1,24 +1,35 @@
...
 
-The [prerelease workflow](https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/prerelease.yml#L11-L46) will update the [SpecsTesting repo](https://github.com/firebase/SpecsTesting) nightly from the head of the master branch.
-In order to let presubmit tests run on the latest podspec repo, [SpecsTesting repo](https://github.com/firebase/SpecsTesting) will be updated when a PR with changed podspecs is merged.
-When this PR is merged, changed pospecs will be `pod repo push`ed to the podsepc repo in [postsubmit tests](https://github.com/firebase/firebase-ios-sdk/blob/master/.github/workflows/prerelease.yml#L48-L94).
+The [prerelease workflow](https://github.com/firebase/firebase-ios-sdk/blob/master/.github/
+workflows/prerelease.yml#L11-L46) will update the [SpecsTesting repo](https://github.com/firebase/
+SpecsTesting) nightly from the head of the master branch.
+In order to let presubmit tests run on the latest podspec repo, [SpecsTesting repo](https://github.
+com/firebase/SpecsTesting) will be updated when a PR with changed podspecs is merged.
+When this PR is merged, changed pospecs will be `pod repo push`ed to the podsepc repo in [
```
Comment (paulb777) [inline scripts/create_spec_repo/README.md]: s/pospecs/podspecs/
s/podsepc/podspec/

**Review by paulb777 — APPROVED** [review]
LGTM with two spelling fixes

**Pull request merged**